### ISSUE TITLE
It takes too long to load

### ISSUE THREAD

**Issue by 45Green** [EFForg/https-everywhere#13546]
Type: other
Domain: all domains.

This extension takes too long to load on page loading.
When I disable it, the pages has load much faster.

**Comment by bardiharborow** [comment]
@45Green is this a specific issue with one website, or a general issue? What extention version, browser version, OS version and hardware are you running? Is your machine CPU, I/O or memory constrained? Are you able to provide benchmarks?

**Comment by 45Green** [comment]
@bardiharborow 
Extension version: 2017.10.30
Browser version: 61.0.3163.100 (stable / 64-bit) 
OS: Windows 10 (version: 1703)
Hardware:
- Intel(R) Core(TM) i5-2430M CPU @ 2.40GHz
- 6 GB DDR3
Pages hangs to load as "waiting on HTTPS everywhere" displays at the bottom of my screen.
I can't provide properly benchmarks:
- Enabled: 3-5 ( 1-2 with cache )
- Disabled: <1 ( <1 with cache )

**Comment by Bisaloo** [comment]
Do you have an example webpage where this problem is easily observable? I can't reproduce when I try to visit random websites covered by HTTPS-Everywhere.

Two points to note:
- I am not sure if chrome notification "waiting on HTTPS Everywhere" is really reliable.
- The fact that it loads faster when you disable HTTPS Everywhere may in part be due to the fact that some assets, DNS records, etc. are now cached.

**Comment by pipboy96** [comment]
Duplicate of [lots of issues].
